SurveyApp/Application/AppDelegate/AppDelegate.swift

```diff
@@ -14,6 +15,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        NetworkActivityLogger.shared.startLogging()
```

Comment:
Please ignore logging from release build by using
```swift
#if DEBUG
```
or using your environment variable 👍 

Btw, you may want to set `debug` level to have a better log message
```swift
NetworkActivityLogger.shared.level = .error
```

Reply:
Oh thank you 🙆‍♀️  I updated that.